taas/jsonld-contexts/taas.jsonld

```diff
@@ -1,5 +1,6 @@
 {
   "@context": {
-    "name": "https://schema.org/name"
+    "name": "https://schema.org/name",
+    "description": "http://purl.org/dc/terms/description"
```

Comment:
it is already in the core context, not sure you need to add it in a business context

Reply:
I still have problem with name and description on TaaS but I don't want to touch anything since the TaaS is very sensitive. I've put it back on core context and description doesn't seem to work. But if I use the taas compound context the problem seems to be worse. An other strange thing is that the name seemed to be working in November but not anymore... Context hasn't changed...
![image](https://user-images.githubusercontent.com/36960251/216949427-1647e4d8-a4e0-441c-aa00-3153d08cb805.png)

Anyway, I learned my lesson the hard way about context with Adeptness :p